transfuse-core/src/main/java/org/androidtransfuse/gen/FilerSourceCodeWriter.java

```diff
@@ -62,9 +61,6 @@ private String toQualifiedClassName(JPackage pkg, String fileName) {
 
     @Override
     public void close() throws IOException {
-        for (OutputStream openStream : openStreams) {
```

Comment:
i remove this because the outputstream already closed an causing error. From the log i think it already calling flush and close, do you have any concern if  we remove this ? i also need to adjust test because of this